### DISCUSSION TITLE
External Bias T

### DISCUSSION THREAD

**Discussion by aclel** [tomojitakasu/PocketSDR#27] [Q&A] [answered]
Am I correct in saying the 4CH model requires an external Bias T? I can see 2CH v2.2 had a Bias T, but it looks to be removed on 2CH v2.3 and 4CH.

Do you have any recommendations for a Bias T? My antenna is [this one](https://www.ardusimple.com/wp-admin/admin-post.php?action=generate_product_pdf&product_id=20982).

**Comment by Data2Logic** [comment]
From the schematic, it's look like it already has bias tee there. I think it should be good enough, but let's wait for author reply
![image](https://github.com/user-attachments/assets/85247856-89e3-4119-b7cf-dd00b2b50907)

**Comment by tomojitakasu** [comment, marked as answer]
Pocket SDR FE 2CH v.2.3 and FE 4CH already have bias-tees.

Pocket SDR FE 2CH v.2.1 and v.2.2 also have bias-tees but it uses a little expensive Mini-Circults TCBT-2R5G+. I replaced these bias-tees by descrete parts to reduce the cost in FE 2CH v.2.3 and FE 4CH.

The antenna DC voltage is 4.2 - 5.0V and the max current is 100 mA with short-circuit protection.
